cli.ts

```diff
@@ -541,9 +541,13 @@ const executingBuiltInTask = (inputArgs: SanitizedInitArgs | RawInitArgs) =>
             
 export const run = (env: EnvVars, inputArgs: DenoArgs, i18n: i18n) => {
     try {
-        // A hack to get around yargs because it strips leading and trailing double quotes of strings passed by the command
-        // Refer to https://github.com/yargs/yargs-parser/issues/201
-        inputArgs = inputArgs.map(arg => arg.match(/^"(.|\n)*"$/) ? "___" + arg + "___" : arg)
+        inputArgs = inputArgs.map(arg => {
```

Comment:
Would you mind moving these hacks to a separate function? Perhaps call it `preprocessArgs` or something like that.